packages/opensrp-store/src/store/store.ts

```diff
@@ -10,21 +10,14 @@ const defaultReducers: Registry = {
   router: connectReducer as Reducer<RouterState>,
 };
 
-// Grab the state from a global variable injected into the server-generated HTML
-const preloadedState = window.__PRELOADED_STATE__ || {};
-// Allow the passed state to be garbage-collected
-delete window.__PRELOADED_STATE__;
```

Comment:
Why did we remove this?

Reply:
@kelvin-muchiri was this one addressed?

Reply:
@moshthepitt I don't know why we removed it. I'll add it back